.github/workflows/build.yml

```diff
@@ -29,11 +29,11 @@ jobs:
       - name: Compile
         run: |
           go mod download
-          cd cmd/junod
+          cd cmd/Junod
```

Comment:
why `J` instead of `j`

Reply:
consistency:

Repo name is:

github.com/CosmosContracts/Juno

Reply:
but heh, seems everywhere else, it's lowercase.  

:P

I'll lowercase this one for consistency

:P